x/bank/simulation/proposals_test.go

```diff
@@ -20,8 +20,7 @@ func TestProposalMsgs(t *testing.T) {
 	s := rand.NewSource(1)
 	r := rand.New(s)
 
-	ctx := sdk.NewContext(nil, true, nil)
-	accounts := simtypes.RandomAccounts(r, 3)
+	_ = simtypes.RandomAccounts(r, 3)
```

Comment:
can this be removed?

Reply:
There is some magic happening with rand there as its Read() fn its being used inside `RandomAccounts`. If removed the simulated params change completely in all tests.